### ISSUE TITLE
Doesn't run on PHP8.1 and Laravel 8

### ISSUE THREAD

**Issue by binaryk** [facade/ignition#430]
It blocks the request on PHP8.1: 
![CleanShot 2021-11-29 at 14 26 19](https://user-images.githubusercontent.com/6833714/143867955-691cf032-41ba-4ea9-b7eb-185cd77cba1d.png)

I have a feeling it tries to boot the ignition provider too early.

Ignition version: 2.17.1
Laravel: 8.73.2
PHP: 8.1.0

**Comment by binaryk** [comment]
Oh, forgot to mention, it's related to the `flare` integration.

**Comment by freekmurze** [comment]
We'll take a look at it shortly. If you already know a good solution, feel free to send a PR.

**Comment by binaryk** [comment]
The issue is here, when `IgnitionServiceProvider` tries to check the log singleton it fails: 
![image](https://user-images.githubusercontent.com/6833714/143876160-2438466f-9bce-4643-8fc7-b7a208981665.png)

I guess it's because the Ignition provider loads too early :( Still looking into it.

**Comment by rubenvanassche** [comment]
Hi @binaryk,

Could you share some more lines of the stack trace? And tell me how you exactly created this exception? Since I cannot recreate the issue locally with a fresh Laravel installation with the version numbers you provided.

**Comment by binaryk** [comment]
It's very weird @rubenvanassche, because many hours of debugging, I finally removed the `barryvdh/laravel-debugbar` and opened the `app.php` boostrap file (and autoload.php file) and then it started working: 
![image](https://user-images.githubusercontent.com/6833714/143892814-059f3431-4fe2-4019-a36a-836821875e87.png)

I reseted my changes, and reinstalled everything again using composer and it started working! This is completely weird :) Basically the `class_exists(LogManager::class)` had false before I've opened bootstrap files (I can guess it was cached or something). 

I'll reopen it with more details if the issue arise again. 

Thank you!
